miso-service/src/main/java/uk/ac/bbsrc/tgac/miso/service/security/DefaultAuthorizationManager.java

```diff
@@ -130,9 +130,13 @@ public void throwIfNotWritable(SecurableByProfile resource) throws IOException,
     return filtered;
   }
 
+  /**
+   * Does not throw if owner is null (editable by anyone).
+   */
   @Override
   public void throwIfNonAdminOrMatchingOwner(User owner) throws IOException, AuthorizationException {
     User currentUser = getCurrentUser();
+    if (owner == null) return;
```

Comment:
This is an unintended grant of permissions to all users. If the owner doesn't exist, this should instead call throwIfNonAdmin